assets/javascripts/discourse/services/chat.js

```diff
@@ -319,6 +319,14 @@ export default Service.extend({
         this.currentUser.chat_channel_tracking_state[a.id]?.unread_count || 0;
       const unreadCountB =
         this.currentUser.chat_channel_tracking_state[b.id]?.unread_count || 0;
+
+      const userAIsSelf = a.chatable.users[0].id === this.currentUser.id;
+      const userBIsSelf = b.chatable.users[0].id === this.currentUser.id;
```

Comment:
I don't think your personal DM channel should be at the top of the list always, other chat apps don't do this that I've seen. Was this just something you thought would be good, or have you seen this elsewhere?

Reply:
Slack does that! 🤔

Reply:
Leaving this as-is, since we talked about it in chat. :p

Reply:
I think I'll avoid messaging myself until there's the ability to close a conversation 😅 I don't want to be stuck with least relevant chat at the top! 😛